# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/root-tail/comments.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur root-tail-1.2.orig/config.h root-tail-1.2/config.h
--- root-tail-1.2.orig/config.h	2004-07-30 21:32:39.000000000 +0200
+++ root-tail-1.2/config.h	2004-11-09 00:04:42.000000000 +0100
@@ -4,10 +4,10 @@
 #define DEF_CONT_COLOR  "grey"
 
 
-//default font.. -font at runtime should work
+/* default font.. -font at runtime should work */
 #define USE_FONT        "*"
 
-//default positions.. can be changed with -g at runtime
+/* default positions.. can be changed with -g at runtime */
 #define STD_WIDTH       730
 #define STD_HEIGHT      530
 #define LOC_X           30
